acme/setup.py

```diff
@@ -26,6 +27,15 @@
     'six>=1.9.0',  # needed for python_2_unicode_compatible
 ]
 
+setuptools_known_environment_markers = (StrictVersion(setuptools_version) >= StrictVersion('36.2'))
+if setuptools_known_environment_markers:
+    install_requires.append('mock ; python_version < "3.3"')
+elif sys.version_info < (3,3):
+    install_requires.append('mock')
+else:
+    raise RuntimeError('Error, you are trying to build certbot wheels using an old version '
```

Comment:
Any reason not to use the same logic as [here](https://github.com/certbot/certbot/blob/e9895d2ec69395cfcf1cfff35e7668b57e1da00f/certbot/setup.py#L59-L74)? With the code in this PR, I think if we try to build the wheel with Python < 3.3 (which [we currently do](https://github.com/certbot/certbot/blob/8e4dc0a48c6108311cf68ca1c683d29b15d49c97/tools/_release.sh#L62)) and an old version of `setuptools`, the resulting wheel won't specify the dependency correctly.

Reply:
wait, isn't this is the opposite of "I personally think we should add mock and not error out in the case of sys.version_info < (3,3) and old setuptools when we're not building a wheel."?

Reply:
Maybe you meant specifically the `bdist_wheel` thing.